website/src/content/docs/docs/howtos/Generate with tsp-client/tsp_client.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: tsp-client usage
+title: Commands
```

Comment:
Sorry reopening as my reply might have got lost, I think actually this name is not really accurate 
<img width="289" alt="image" src="https://github.com/user-attachments/assets/6ff27171-b558-482b-af98-06e271d0f5f1">

I think either we should split this file or maybe find a better name

Reply:
Ok I dont want to do the refactoring for the other doc in this PR, but I agree I think we can find a way to split it. Maybe between usage & configuration (dev), commands and configurations, configuration (repo). 

For this PR I could revert the renaming we just did or update it to `Usage & configuration` which I think covers most of the content of the current documentation, what do you think?

Reply:
however you feel, just depends I think on when the follow up will be done

Reply:
`Usage & Configuration` dos also seems like a good name without needing a split

Reply:
```
tsp-client
  Quick Start
  Usage & Configuration
```

Reply:
Agree will update to those names for now, and I opened this tracking issue to look into splitting the docs: https://github.com/Azure/azure-sdk-tools/issues/9357